build/package/download-win-binaries.sh

```diff
@@ -30,7 +30,15 @@ if [[ "${TOOL_NAME:?}" == atlascli ]]; then
 	PACKAGE_NAME="mongodb-atlas-cli_${VERSION_NAME}_windows_x86_64.msi"
 fi
 
+BINARY_NAME="mongocli.exe"
+if [[ "${TOOL_NAME:?}" == atlascli ]]; then
+	BINARY_NAME="atlas.exe"
+fi
+
```

Comment:
[nit] this does not need another if. you can add `BINARY_NAME="atlas.exe"` inside the previous if. Similarly, you can move `BINARY_NAME="mongocli.exe"` at line 27

Reply:
I noticed too late, pushed a fix